<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PicViewerClass</class>
 <widget class="QWidget" name="PicViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PicViewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MilesEdgeworth.qrc">
    <normaloff>:/icon/prosbadge_icon.png</normaloff>:/icon/prosbadge_icon.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="MyGraphicsView" name="graphicsView">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actOpen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>添加图片</string>
   </property>
   <property name="toolTip">
    <string>选择图片文件添加</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actFull">
   <property name="text">
    <string>显示原图大小</string>
   </property>
   <property name="toolTip">
    <string>显示为原图大小</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actFit">
   <property name="text">
    <string>图片适应窗口</string>
   </property>
   <property name="toolTip">
    <string>显示为适应屏幕大小</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actAdjust">
   <property name="text">
    <string>窗口适应图片</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">MyGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MilesEdgeworth.qrc"/>
 </resources>
 <connections/>
</ui>
